Documentation/performance-profiling-setup-test.md

```diff
@@ -0,0 +1,29 @@
+# Set up a repeatable test
+
+We need an area of code to execute repeatedly, with as many variables locked down as possible. 
+
+
+### Set up Unreal
+1) Open Unreal Editor (UnrealEditor.exe)
+2) Create a blank map (project doesn't matter. Choose an existing one or create a new one)
+3) Go to Edit->Plugins
+4) Search for "Functional Testing plugin". Check it to enable it
+![smaller](https://github.com/CesiumGS/cesium-unreal/assets/130494071/5a3bc9de-cdaf-4d9d-842d-104719426663)
+5) Save all
+6) Set this map as the 'Editor Startup Map' so it loads when starting from Visual Studio
+![smaller 2](https://github.com/CesiumGS/cesium-unreal/assets/130494071/8ba5c6c2-8c97-4048-afe2-db74770d85cc)
+
+
+### Build Release Code
+
+We need to make sure all our C++ code is building in release mode.
+
+> This assumes that you have already built your code successfully and are familiar with the concepts from our [developer setup guide](https://github.com/CesiumGS/cesium-unreal/blob/ue5-main/Documentation/developer-setup-windows.md). Although you could profile a debug build, it is typically more useful to build in release, since this is how a game is usually packaged.
+
+1) If building the cesium-native library, make sure you are using a Release configuration
```

Comment:
Preferably `RelWithDebInfo` rather than `Release`, so that you get symbol names, right?

Reply:
Hmm strange, in VS Code the configurations look like this:
<img width="453" alt="image" src="https://github.com/CesiumGS/cesium-unreal/assets/924374/ea17d48d-a0fd-462b-8fc3-9d76b5cf28ad">

And I believe these match up with the CMake configurations. Visual Studio must be doing its own thing.

Reply:
Yes, you are correct. Visual Studio is making its own configuration for release, but defaults to the cmake config RelWithDebugInfo. So that's good.

I've updated the documentation to note this preference and how this config is selected by default in VS.